Rakefile

```diff
@@ -33,54 +51,67 @@ def bump_to_version(from, to)
   spec_path = "xcproj.podspec"
   content = File.read(spec_path)
   File.open(spec_path, "w"){|f| f.write(content.sub(from.to_s, to.to_s)) }
+end
+
+def pod_lint
+  sh "bundle exec pod install --project-directory=CocoaPods/"
+  sh "xcodebuild -workspace CocoaPods/CocoaPods.xcworkspace -scheme macOS -config Debug clean build"
+  sh "xcodebuild -workspace CocoaPods/CocoaPods.xcworkspace -scheme iOS -config Debug -destination '#{DESTINATION}' clean build"
+end
+
+def commit_changes_and_push(tag)
   `git add .`
   `git commit -m "Bump version to #{to}"`
-  `git tag #{to}`
+  if tag then `git tag #{tag}` end
   `git push origin --tags`
 end
 
-def print(message)
-  puts message.colorize(:yellow)
+def generate_carthage_project
+  throw "Mint is necessary. Make sure it's installed in your system" unless command?("mint")
+  sh "mint run yonaskolb/xcodegen@1.3.0"
```

Comment:
😍 👍

Reply:
That's a good idea @esttorhe. However, I tried to do it but it's not supported by Mint yet. When you pass a parameter to specify the project using mint, mint tries to parse that parameter. I've opened an [issue](https://github.com/yonaskolb/Mint/issues/13) to track it. Do you know if we could do this somehow or we need to add support for that in Mint @yonaskolb ?